bin/oref0-online.sh

```diff
@@ -2,13 +2,17 @@
 MAC=$1
 echo -n "At $(date) my IP is: "
 if ! curl -m 15 icanhazip.com; then
-    echo -n "Error, cycling wlan0 "
-    sudo ifdown wlan0
-    sudo ifup wlan0
-    echo -n "and getting new wlan0 IP"
-    ps aux | grep -v grep | grep -q "dhclient wlan0" && sudo killall dhclient
-    sudo dhclient wlan0 -r
-    sudo dhclient wlan0
+    echo -n "Error, cycling networking "
+   # sudo ifdown wlan0
+  #  sudo ifup wlan0
+  #  echo -n "and getting new wlan0 IP"
```

Comment:
as far as I know it stops everything - Ive never had to restart DHCP but its a fair question - is there a way to confirm?